services/api-server/src/simcore_service_api_server/api/routes/studies.py

```diff
@@ -85,9 +87,14 @@ async def get_study(
 async def clone_study(
     study_id: StudyID,
     webserver_api: Annotated[AuthSession, Depends(get_webserver_session)],
+    x_simcore_parent_project_uuid: Annotated[ProjectID | None, Header()] = None,
+    x_simcore_parent_node_id: Annotated[NodeID | None, Header()] = None,
```

Comment:
Q: out of curiosity, why via request headers?

Reply:
My interpretation is that it provides context/metadata about the request and therefore it makes sense to go in the header. E.g. in the header you can find the server type, etc...

Reply:
Q: in the osparc client, perhaps we should have these captured from the env vars by default e.g. we can connect to the env-vars defined in jupyter etc

Reply:
Yep, that's exactly the plan. 😄